modules/log/src/EventSubscriber/OrderEventSubscriber.php

```diff
@@ -34,6 +36,7 @@ public static function getSubscribedEvents() {
       'commerce_order.validate.pre_transition' => ['onValidateTransition', -100],
       'commerce_order.fulfill.pre_transition' => ['onFulfillTransition', -100],
       'commerce_order.cancel.pre_transition' => ['onCancelTransition', -100],
+      OrderEvents::ORDER_ASSIGN => ['onOrderAssign', -100],
```

Comment:
We don't use constants above, so let's not use it here either, to stay consistent. 
Core generally discourages constants and recommends the real IDs to be used.